### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by VedanthB**
- added the wishlist page 
- this is the preview link https://deploy-preview-3--romantic-mccarthy-7219f3.netlify.app/pages/wishlist-page/wishlist-page.html

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -14,6 +14,13 @@
 .bg-regal-blue-dark {
   background-color: #000022;
 }
+.text-regal-blue {
+  color: #2a2840;
+}
+
+.text-regal-blue-dark {
+  color: #000022;
+}
```
Comment (Richa-rani02) [inline index.css:23]: add variable name for colors

**Review by Richa-rani02 — CHANGES_REQUESTED** [review]

File: pages/product-page/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.divider {
+  border: 1px solid var(--regal-blue-dark);
+  width: 80%;
+  margin: auto;
+}
+.filters-sidebar-container {
+  height: 100%;
+  width: 20%;
+  /* border: 1px solid var(--regal-blue-dark); */
```
Comment (Hardiegogo) [inline pages/product-page/product-listing.css:9]: Remove unused code.

**Review by Hardiegogo — COMMENTED** [review]
Looks good.
Just a small change and then you can merge.

File: index.css (hunk 1 above)
Comment (VedanthB) [reply index.css:23]: okay will make these changes , thanks for reviewing the code 😄

File: pages/product-page/product-listing.css (hunk 2 above)
Comment (VedanthB) [reply pages/product-page/product-listing.css:9]: okay will make these changes , thanks for reviewing the code 😄

**Pull request merged**